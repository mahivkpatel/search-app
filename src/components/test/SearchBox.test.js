import React from 'react'

import { shallow } from 'enzyme'
import { SearchBox } from '../search-box/search-box.component'
it('expect render SearchBox component', () => {
  expect(shallow(<SearchBox placeholder="search user" />)).toMatchSnapshot()
})
