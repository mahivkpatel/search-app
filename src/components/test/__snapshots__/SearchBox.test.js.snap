// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render SearchBox component 1`] = `ShallowWrapper {}`;
